File: ts-sdk/src/sdk/models/operations/optionsevents.ts
import { SpeakeasyMetadata, SpeakeasyBase, RetryConfig } from "../../../internal/utils";
import * as shared from "../shared";



export class OptionsEventsRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  retries?: RetryConfig;
}


export class OptionsEventsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  clientError?: shared.ClientError;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  realtimeServers?: shared.RealtimeServers;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: go-sdk/pkg/models/operations/optionsevents.go
package operations

import (
	"openapi/pkg/models/shared"
	"openapi/pkg/utils"
)

type OptionsEventsRequest struct {
	Retries *utils.RetryConfig
}

type OptionsEventsResponse struct {
	ClientError     *shared.ClientError
	ContentType     string
	RealtimeServers *shared.RealtimeServers
	StatusCode      int64
}
